docs/sources/explore/_index.md

```diff
@@ -59,6 +59,46 @@ In split view, timepickers for both panels can be linked (if you change one, the
 
 To close the newly created query, click on the Close Split button.
 
+## Share Explore URLs
+
+When using Explore, the URL in the browser address bar is updated as you make changes to the queries. This allows you to share the URLs with others or bookmark them for later use.
+
+### Generating Explore URLs from external tools
+
+Because Explore URLs have a defined structure, you can build an URL from external tools and open it in Grafana. The URL structure is:
+
+```
+http://<grafana_url>/explore?panes=<panes>&schemaVersion=<schema_version>&orgId=<org_id>
+```
+
+where:
+
+- `org_id` is the organization ID
+- `schema_version` is the schema version, currently only `1` is supported
+- `panes` is a url-encoded JSON object of panes, where each key is the pane ID and each value is an object matching the following schema:
+
+```ts
+{
+  datasource: string; // the datasource UID, or `-- Mixed --` for mixed datasources
+  queries: {
+    // the queries for this pane
+    refId: string; // an alphanumeric identifier for this query, must be unique within the pane, i.e. "A", "B", "C", etc.
+    datasource: {
```

Comment:
Is it actually required when a non-mixed data source is used at the root? If not maybe we can mention it, otherwise it might be confusing why it needs to be set up twice when non-mixed is used.

Reply:
not strictly required as Explore will use the current root datasource if the query doen't have the datasource property, but that was done for legacy reasons. when not defining a datasource in each query explore will update the URL to include on for each, so effectively the URL will match this schema, not one without that property

Reply:
Then maybe we could add a bit more context for both, e.g.:

```typescript
datasource: string; // Explore's root datasource UID, or `-- Mixed --` for mixed datasources
...
uid: string; // query's datasource UID ie: "AD7864H6422"
```

Reply:
updated in d05619da00